lib/Froxlor/Api/Commands/Domains.php

```diff
@@ -312,6 +314,7 @@ public function add()
 				$documentroot = $this->getParam('documentroot', true, '');
 				$phpenabled = $this->getBoolParam('phpenabled', true, 0);
 				$openbasedir = $this->getBoolParam('openbasedir', true, 0);
+				$openbasedir_path = $this->getBoolParam('openbasedir_path', true, 0);
```

Comment:
`getBoolParam()` will only give your values `1` or `0` never the value `2` which is also possible for this setting

Reply:
Fixed